src/wallet/wallet2.cpp

```diff
@@ -3502,6 +3502,9 @@ uint64_t wallet2::import_key_images(const std::vector<std::pair<crypto::key_imag
     req.key_images.push_back(epee::string_tools::pod_to_hex(key_image));
   }
 
+  for (size_t n = 0; n < signed_key_images.size(); ++n)
```

Comment:
The single line for without {}s seems inconsistent with the coding "standards" in other parts of the project and can introduce bugs later if someone is reading quickly and just inserts additional lines of code without verifying the {}s.